internal/handlers/auth_oidc.go

```diff
@@ -87,7 +89,7 @@ func authLoginOidcCallbackHandler(w http.ResponseWriter, r *http.Request) {
 		} else if len(orgs) < 1 {
 			// TODO deduplicate (regular login)
 			org.Name = user.Email
-			org.UserRole = types.UserRoleVendor
+			org.UserRole = env.DefaultUserRole()
```

Comment:
I struggle to imagine a situation where it would make sense to change the default role to `customer`. The user would end up in an empty organization where they can't really do anything.

Is there a reason this was added that I'm missing?

Reply:
No, this is a leftover from something i was testing, and can be discarded